### PR TITLE
Autosave status

### PR DESCRIPTION
**Pull request by HbsRed**

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -693,6 +693,7 @@
 			stat("Local Time", stationtime2text())
 			stat("Local Date", stationdate2text())
 			stat("Server Uptime", roundduration2text())
+			stat("Autosave Timer"), SSautosave.time_next_save
```
Comment (PsyCommando) [inline code/modules/mob/mob.dm:696]: The parenthesis at the end is missing. Also, that would display time in deciseconds, which would be a huge meaningless number to most people.

**Review by PsyCommando — CHANGES_REQUESTED** [review]
You didn't compile this or even test it did you? :/

**Comment by HbsRed** [comment]
See thing is.. i follow brawlers good ol advice and hes like yup sounds
great then sha bang. It no work pepe but you see what im trying to do yeah?

**Comment by PsyCommando** [comment]
> 
> See thing is.. i follow brawlers good ol advice and hes like yup sounds great then sha bang. It no work pepe but you see what im trying to do yeah?
> […](#)

Well I mean, my main concern is having the code work, so the server works XD Which is why I'm requesting changes on this pr so it can still work if this is merged XD

**Comment by PsyCommando** [comment]
I guess I'll close this one, since you don't seem interested in fixing the PR.

**Pull request closed without merging**